pkg/collector/corechecks/cluster/kubernetesapiserver/unbundled_events.go

```diff
@@ -79,6 +80,10 @@ func (c *unbundledTransformer) Transform(events []*v1.Event) ([]metrics.Event, [
 
 		emittedEvents.Inc(involvedObject.Kind, ev.Type)
 
+		for _, t := range config.GetConfiguredTags(config.Datadog, false) {
```

Comment:
This should probably come before call to `SortUniq`.

```suggestion
		for _, t := range config.GetGlobalConfiguredTags(false) {
```